permissions/plugin-pipeline-milestone-step.yml

```diff
@@ -4,4 +4,9 @@ paths:
 - "org/jenkins-ci/plugins/pipeline-milestone-step"
 developers:
 - "amuniz"
+- "svanoort"
+- "abayer"
 - "jglick"
+- "rsandell"
+- "stephenc"
+- "oleg-nenashev"
```

Comment:
And Oleg is `oleg_nenashev`.

Sooooo many existing YAML files to copy user names from…